Mathlib/Analysis/InnerProductSpace/Dual.lean

```diff
@@ -174,7 +174,8 @@ variable (B : E →L⋆[𝕜] E →L[𝕜] 𝕜)
 
 @[simp]
 theorem continuousLinearMapOfBilin_apply (v w : E) : ⟪B♯ v, w⟫ = B v w := by
-  simp [continuousLinearMapOfBilin]
+  rw [continuousLinearMapOfBilin, coe_comp', ContinuousLinearEquiv.coe_coe,
+  LinearIsometryEquiv.coe_toContinuousLinearEquiv, Function.comp_apply, toDual_symm_apply]
```

Comment:
```suggestion
    LinearIsometryEquiv.coe_toContinuousLinearEquiv, Function.comp_apply, toDual_symm_apply]
```

Reply:
This is just `rfl` though, right?

Reply:
No, `toDual_symm_apply` is far from rfl.

Reply:
Fixed the spacing